### DISCUSSION TITLE
Is there any Spotfire Mod version of LineUp?

### DISCUSSION THREAD

**Discussion by heberleh** [lineupjs/lineupjs#617] [Q&A]
Now that Mods empowers users to create custom visualizations in Spotfire more officially and even deploy them in the Spotfire Library, I wonder if someone has started to code such Mod for LineUp.

Any idea? Thanks!

**Comment by mstreit** [comment]
Not yet, but it would be nice to have an implementation in the store. Any volunteers? :)

**Reply by heberleh** [reply]
I'll spread the call internally, just in case there is someone else interested too.

**Reply by heberleh** [reply]
Wondering how PSE could enter the boat too.
